moveit_ros/moveit_servo/src/servo_calcs.cpp

```diff
@@ -982,6 +1026,42 @@ bool ServoCalcs::getCommandFrameTransform(Eigen::Isometry3d& transform)
   return !transform.matrix().isZero(0);
 }
 
+bool ServoCalcs::getCommandFrameTransform(geometry_msgs::TransformStamped& transform)
+{
+  const std::lock_guard<std::mutex> lock(latest_state_mutex_);
+  // All zeros means the transform wasn't initialized, so return false
+  if (tf_moveit_to_robot_cmd_frame_.matrix().isZero(0))
```

Comment:
Shouldn't we initialize the matrix as zero matrix explicitly for this to work reliably with all compiler settings?

Reply:
good idea, done.